<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveProjectionForm</class>
 <widget class="QWidget" name="SaveProjectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save projections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0">
   <item row="0" column="0">
    <widget class="QGroupBox" name="typeGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>File type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="projectionRadio">
        <property name="toolTip">
         <string>Writes in file projection from some selected respectived node.</string>
        </property>
        <property name="text">
         <string>Projections</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="shortPathRadio">
        <property name="toolTip">
         <string>Writes in file all available short paths from one node to another.</string>
        </property>
        <property name="text">
         <string>Short paths</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Projection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="printIndentsCheck">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Print indents for more readable data. Used in bracket style.&lt;br/&gt;
Each new nesting increase indent lewel.&lt;/p&gt;
&lt;p&gt;
Example:
&lt;table&gt;
&lt;tr&gt;
&lt;td&gt;
Turned off:&lt;br&gt;
1(2(3(4 5) 7))
&lt;/td&gt;
&lt;td&gt;
Turned on:&lt;br&gt;
1(&lt;br&gt;
&amp;middot; &amp;middot; 2(&lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; 3(&lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; 4&lt;br&gt; 
&amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; &amp;middot; 5&lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; ) &lt;br&gt;
&amp;middot; &amp;middot; &amp;middot; &amp;middot; 7&lt;br&gt;
&amp;middot; &amp;middot; )&lt;br&gt;
)
&lt;/td&gt;
&lt;/tr&gt;
&lt;/table&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Print indents</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="printInfoCheck">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prints graph additional information. &lt;/p&gt;&lt;p&gt;For example: &lt;br/&gt;name of generated type (hypercube torus); &lt;br/&gt;parameters, used by generator (dimention, big raidus, small radius);&lt;br/&gt;basic weight of all edges;&lt;/p&gt;&lt;p&gt;Example in file:&lt;/p&gt;&lt;p&gt;{TYPE=HYPERCUBE}&lt;br/&gt;{DIMENTION=6}&lt;br/&gt;{WEIGHT=1}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Print graph info</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="printAllNodesCheck">
        <property name="text">
         <string>Print all nodes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reverseCheck">
        <property name="text">
         <string>In reverse</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start node</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="startNodeLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Node id for start writing bracket style paths.</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Second node</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="secondNodeLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Example</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="exampleLabel">
        <property name="text">
         <string notr="true">example</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
